Sources/Networking/AlamofireNetworkClient/AlamofireConsoleLogger.swift

```diff
@@ -14,7 +14,7 @@ public final class AlamofireConsoleLogger: EventMonitor {
   public let queue = DispatchQueue(label: "com.alamofire.console.networklogger")
   private let logger: (String) -> Void
   
-  public init(logger: @escaping (String) -> Void = { Logger.debug($0) }) {
+  public init(logger: @escaping (String) -> Void = { Logger.debug($0, line: -1) }) {
```

Comment:
Is this intentional or is it a leftover?

Reply:
That is intentional. If you take a look at [this line](https://github.com/poviolabs/PovioKit/pull/295/files#diff-e17de7f55086084fc2cab7b1efeeea7e81eea4033353c1af71c92b0e4e53ddc9R77), you'll notice we don't output source metadata when line is below 0. In this case we try to omit source output since it's always pointing to the `AlamofireConsoleLogger`, thus irrelevant. Hope it makes sense. If you have a better suggestion, lmk.

Reply:
I see, it makes sense...